WIP: move talks to a new page #712 (#713)
buildpacks/docs: content/docs/talks.md

+++
title="Talks"
weight=2
getting-started=true
+++

## Talks

We love talks to share the latest development updates, covering Buildpacks basics and more, receive feedback and questions, and get to know other members of the community.

> ### Introductory Talks  
> 
> If you are new to the project, you can watch some of introductory talks on [Cloud Native Buildpacks' YouTube channel](https://www.youtube.com/@buildpacks)

Reviewer: ```suggestion
```

We have not been very good stewards of our YouTube channel. I would drop the meeting recordings below the conference talks to increase the chances that the first click will be useful.

> ### Conference Talks
> 
> {{< talks >}}
> 
> More conference talks are available under the [Conference Talks Playlist](https://www.youtube.com/playlist?list=PL1p8pquzNvRqz3v7Q-OA7wldIxLCNFHcC) on YouTube.  

Reviewer: ```suggestion
### Conference Talks

Check out some of our most recent and exciting conference talks below. More talks are available in our [Conference Talks Playlist](https://www.youtube.com/playlist?list=PL1p8pquzNvRqz3v7Q-OA7wldIxLCNFHcC) on YouTube.

{{< talks >}}

If you are interested in giving a talk about buildpacks, the linked slides may provide a useful starting point. Please feel free to reach out in Slack if you'd like input or help from the CNB team!
```

It's a bit weird that the `{{< talks >}}` block also drops in a "Resources" with downloadable slides. It makes the "More conference talks are available..." text seem a bit out of place. WDYT?

> ### Working Group Meetings
> 
> Feel free to look through the archive of previous community meetings on our [Working Group YouTube Playlist](https://www.youtube.com/playlist?list=PL1p8pquzNvRpDbbgZ0db0MRA-W5_w0G1U)

Reviewer: ```suggestion
### Meetings

Feel free to look through the archive of previous community meetings in our [Working Group Playlist](https://www.youtube.com/playlist?list=PL1p8pquzNvRpDbbgZ0db0MRA-W5_w0G1U) on YouTube. If you would like to attend a Working Group meeting, check out our [community](/docs/community) page.
```
